simplified-app-shared/src/main/java/org/nypl/simplified/app/Simplified.kt

```diff
@@ -17,6 +19,8 @@ import org.nypl.simplified.observable.ObservableReadableType
 import org.nypl.simplified.prefs.Prefs
 import org.slf4j.LoggerFactory
 import java.io.File
+import java.io.IOException
+import java.util.*
```

Comment:
:nauseated_face:

Reply:
wildcard import?

Reply:
Android Studio is great at sneaking in those star imports...